tienvx/mbt-api-bundle/1.0/config/packages/tienvx_mbt_api.yaml

```diff
@@ -0,0 +1,4 @@
+tienvx_mbt_api:
+    java_binary: '%env(JAVA_BINARY)%'
+    plantuml_binary: '%env(PLANTUML_BINARY)%'
```

Comment:
You should help the user by defining these environment variables in your manifest.